explainer/Explainer.md

```diff
@@ -19,12 +21,12 @@ cannot be modified and so can only be described externally.
 
 The WoT Discovery specification attempts to meet these challenges by defining
 a multistage discovery process with authorization checkpoints to meet privacy and
-security goals; by using scalable RESTful API designs; by supporting a variety of
+security goals; by using scalable network API designs; by supporting a variety of
 first-contact mechanisms suitable for different scales; and by providing optional
 semantic search and self-description capabilities.
 
 There are still a number of limitation in the current specification, which
-primarily focuses on HTTP and had to defer some important goals, such as
+primarily focuses on CoAP and HTTP and had to defer some important goals, such as
```

Comment:
As the specification currently focuses on HTTP and in order to be consistent with the order in the rest of the document, I think CoAP and HTTP could be swapped here. Other than that, looks good to me :) Thank you, @mmccool!

```suggestion
primarily focuses on HTTP and CoAP, and had to defer some important goals, such as
```